app/assets/stylesheets/mailer.scss

```diff
@@ -0,0 +1,12 @@
+.email-table{
+  width: 540px;
+  padding: 20px;
+  background: #fff;
+  margin: 40px auto;
+  text-align: left;
+  font-family: Gill Sans Extrabold, sans-serif;
+  border: 5px solid;
+}
+th, td {
```

Comment:
em.. scss tree styling? instead of global?

Reply:
@romaslmd  this is another stylesheet for mailer only, so maybe everything is fine.
Of I don't understand something.
Or you want to put th, td inside email-table?

Reply:
could be
even so, better to avoid such wide-impact styling..